src/SmartComponents/modules/reducers.js

```diff
@@ -17,6 +18,11 @@ function compareReducer(state = initialState, action) {
                 ...state,
                 status: action.payload
             };
+        case `${types.OPEN_ADD_SYSTEM_MODAL}_FULFILLED`:
```

Comment:
You don't need the `_FULFILLED` in this case as that originates from the redux-promise-middleware when making async requests. For this, I think you would just want to reference the action directly that will be dispatched.